python/lsst/ctrl/bps/wms/htcondor/htcondor_service.py

```diff
@@ -1607,3 +1604,57 @@ def _locate_schedds(locate_all=False):
     else:
         schedd_ads.append(coll.locate(htcondor.DaemonTypes.Schedd))
     return {ad["Name"]: htcondor.Schedd(ad) for ad in schedd_ads}
+
+
+def _gather_site_values(config, compute_site):
+    """Gather values specific to given site.
+
+    Parameters
+    ----------
+    config : `lsst.ctrl.bps.BpsConfig`
+        BPS configuration that includes necessary submit/runtime
+        information.
+    compute_site : `str`
+        Compute site name.
+
+    Returns
+    -------
+    site_values : `dict` [`str`, `Any`]
```

Comment:
Please fix the type description.

Reply:
This turns out to not be another mislabeled dict [dict [str Any]]  It's just that there are two specific keys (attrs and profile) that do have dicts for values, but there are other keys that have values of strings, etc.  Leaving the type description as is.